programs/bubblegum/program/tests/simple.rs

```diff
@@ -251,6 +251,26 @@ async fn test_create_public_tree_with_canopy() {
     assert_eq!(cfg.tree_delegate, payer.pubkey());
     assert!(cfg.is_public);
 
+    let tree_create_result = context.create_tree_with_canopy::<19, 64>(1, true).await;
+
+    if let Err(err) = tree_create_result {
+        if let BanksClient(BanksClientError::TransactionError(e)) = *err {
+            assert_eq!(
+                e,
+                TransactionError::InstructionError(0, InstructionError::Custom(6041),)
+            );
+        } else {
+            panic!("Wrong variant");
+        }
+    } else {
+        panic!("Should have failed");
+    }
+}
```

Comment:
Nit: I would actually suggest making this a second test, not part of existing `test_create_poublic_tree_with_canopy()`.

The new test would be called something like `test_cannot_create_tree_needing_too_many_proofs_with_too_small_canopy`